src/event.rs

```diff
@@ -1625,7 +1625,6 @@ mod tests {
 
 		// Check we get the expected event and that it is returned until we mark it handled.
 		for _ in 0..5 {
-			assert_eq!(event_queue.wait_next_event(), expected_event);
```

Comment:
The original idea was to assert that the event can be re-read also via the other method?